### DISCUSSION TITLE
2-byte characters are garbled in the plot

### DISCUSSION THREAD

**Discussion by tasasaki-github** [posit-dev/positron#3803] [Feedback]
Executing the following code will garble 2-byte characters in the plot:

```
ggplot(
  data = penguins,
  mapping = aes(x = bill_length_mm, y = body_mass_g)
  ) +
  geom_point(na.rm = TRUE) + 
  labs(caption = "デ ー タ はpalmerpenguinsパッケージより。")
```

![image](https://github.com/posit-dev/positron/assets/308324/f40873bd-3af2-44db-85ec-a8fb6b948c5a)

I think drawing with AGG (ragg) will solve the garbled characters, so I would like to see a change so that ragg can be used in positron.

**Comment by juliasilge** [comment]
We also would like to use ragg for plot rendering and think that would be the best option in the long term. I added your request to #2919. Thanks!
